src/main/java/com/github/matidominati/medicalclinic/model/entity/Institution.java

```diff
@@ -22,4 +22,21 @@ public class Institution{
     private String address;
     @ManyToMany(mappedBy = "institutions")
     private List<Doctor> doctors;
+    public static Institution create(CreateInstitutionCommand createInstitution) {
+        return Institution.builder()
+                .name(createInstitution.getName())
+                .address(createInstitution.getAddress())
+                .build();
+    }
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        Institution that = (Institution) o;
+        return Objects.equals(id, that.id);
```

Comment:
return id != null Objects.equals(id, that.id);

DO POPRAWY DALEJ ROWNIEZ NIE BEDE JUZ PISAL KOM